Sources/Extensions/SwiftStdlib/OptionalExtensions.swift

```diff
@@ -89,3 +105,4 @@ public extension Optional where Wrapped: Collection {
 
 // MARK: - Operators
 infix operator ??= : AssignmentPrecedence
+infix operator ?= : AssignmentPrecedence
```

Comment:
How does precedence work? If I call
```Swift
a ?= b ?? c
```
will it know to use `??` before `?=` (which is what seems logical to me in this scenario)

Reply:
Sure, it will call `??` before `?=`, because `?=` is AssignmentPrecedence just like the assignment operator (https://developer.apple.com/documentation/swift/swift_standard_library/operator_declarations)

Reply:
Assignment precedence has a value of 90, which is the lowest precedence value of operators in swift. This should mean that it get evaluated last. (I think we can find those values in `import Swift`)